helmfile/values/dex.yaml.gotmpl

```diff
@@ -14,14 +14,19 @@ ingress:
     nginx.ingress.kubernetes.io/whitelist-source-range: {{ .Values.externalTrafficPolicy.whitelistRange.dex }}
     {{ end }}
   tls:
-  - secretName: dex-tls
-    hosts:
-    - dex.{{ .Values.global.baseDomain }}
+    - secretName: dex-tls
+      hosts:
+        - dex.{{ .Values.global.baseDomain }}
   hosts:
-  - dex.{{ .Values.global.baseDomain }}
+    - host: dex.{{ .Values.global.baseDomain }}
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
 
 # https termination by ingress instead of through dex
-https: false
+https:
+  # -- Enable the HTTPS endpoint.
+  enabled: false
```

Comment:
This doesn't seem to be in the new config. Is this the same as `service.ports.https`?

Reply:
It is there, i have just removed it

Reply:
Right, you're correct! I just looked at the file in this repo.

But I'd like if you keep the files in the upstream folder exactly as the upstream repo. That way it's easy to see the upstream defaults etc.

Reply:
solved! :-)